src/routes/Route.tsx

```diff
@@ -54,56 +47,72 @@ const donorOrClientDrawer = () => {
 		QRCodeScannerScreen: {
 			screen: QRCodeScannerScreen,
 			navigationOptions: {
-				drawerLabel: 'Scan QR Code',
-				drawerIcon: DrawerIcon('qrCode'),
+				drawerLabel: <MainMenu text="Scan QR Code" icon="qrCode" />,
 			},
 		},
 		DashboardScreen: {
 			screen: MainStack,
 			navigationOptions: {
-				drawerLabel: 'My Donations',
-				drawerIcon: DrawerIcon('donations'),
+				drawerLabel: <MainMenu text="Donations" icon="claims" />,
+			},
+		},
+		Active: {
+			screen: MainStack,
+			navigationOptions: {
+				drawerLabel: <SubMenu text="Active" />,
+			},
+		},
+		History: {
+			screen: MainStack,
+			navigationOptions: {
+				drawerLabel: <SubMenu text="History" />,
 			},
 		},
 	};
 
 	const CLIENT_MENU = {
+		DonationScreen: {
+			screen: MainStack,
+			navigationOptions: {
+				drawerLabel: <MainMenu text="Donations" icon="donations" />,
+			},
+		},
+
 		DashboardScreen: {
 			screen: MainStack,
 			navigationOptions: {
-				drawerLabel: 'My Claims',
-				drawerIcon: DrawerIcon('claims'),
+				drawerLabel: <MainMenu text="Claims" icon="claims" />,
 			},
 		},
-	};
 
-	const COMMON_MENU = {
-		ProfileScreen: {
+		Pending: {
 			screen: MainStack,
 			navigationOptions: {
-				drawerLabel: 'My Profile',
-				drawerIcon: DrawerIcon('user'),
+				drawerLabel: <SubMenu text="Pending" />,
 			},
 		},
-		SettingsScreen: {
+
+		History: {
 			screen: MainStack,
 			navigationOptions: {
-				drawerLabel: 'Settings',
-				drawerIcon: DrawerIcon('settings'),
+				drawerLabel: <SubMenu text="History" />,
 			},
 		},
+	};
+
+	const COMMON_MENU = {
+
 		HelpScreen: {
 			screen: MainStack,
 			navigationOptions: {
-				drawerLabel: 'Help',
-				drawerIcon: DrawerIcon('help'),
+				drawerLabel: <MainMenu text="Contact Us" icon="help" />,
 			},
 		},
+
 		LogoutScreen: {
 			screen: LogoutScreen,
 			navigationOptions: {
-				drawerLabel: 'Log Out',
-				drawerIcon: DrawerIcon('logout'),
+				drawerLabel: <MainMenu text="Log out" icon="logout" />,
```

Comment:
very very very small thing but the "out" in `Log out` was capitalized before this PR 
:)